### ISSUE TITLE
Error saving files

### ISSUE THREAD

**Issue by Yveltal3434** [juliandescottes/piskel#570]
Piskel wouldn't save my sprite. It wouldn't save to the server, so I downloaded it as a .piskel file. Later inspection found that there was no image data in the file.

This is the formatted JSON inside the .piskel file:
```json
{
    "modelVersion": 2,
    "piskel": {
        "name": "Daddy B-day Card",
        "description": "things",
        "fps": 19,
        "height": 250,
        "width": 400,
        "layers": [
            "{\"name\":\"Layer 1\",\"opacity\":1,\"frameCount\":141,\"base64PNG\":\"data:,\"}"
        ]
    }
}
```

**Comment by juliandescottes** [comment]
Hi @Yveltal3434 ,

From what I can see in the JSON, I think the sprite is just too big for Piskel to save. 
140 frames @ 250x400 seems to be too much. I'll reuse that for the ongoing issue #555 that will show warnings when users might encounter this kind of issues. 

I guess for the .piskel export I can also split it to allow it to be saved (even if it means having a huge file in the end)

For now, if you  what you could try:
- export as GIF (it should be able to handle big sprites)
OR
- delete 1st half of frames -> save -> undo until you have all your frames back -> delete 2nd half of frames -> save (but I would be worried about an application crash here ...)

Sorry about that :/

**Comment by juliandescottes** [comment]
Addressed by #591
